lib/util/print-version.js

```diff
@@ -0,0 +1,6 @@
+const printVersion = () => {
+  let pjson = require('../../package.json');
+  console.log(`Squash version is ${pjson.version}`);
```

Comment:
We want to print just the version squash without anything else.
Let's change this to `console.log(pjson.version)`